doc/ledger3.texi

```diff
@@ -4686,6 +4689,10 @@ normal ledger file (specified via @option{--file @var{FILE} (-f)} or via
 the environment variable @env{LEDGER_FILE}) this entry will not be
 printed again.
 
+As you can see in output above, the account is @samp{Expenses:Unknown}
+for CSV lines. Your can use option @option{--auto-match} to
```

Comment:
Spelling: Change `Your` to `You`
Grammar: Add `the` between `use [the] option`
Wording: Change order of words: `You can use the @option{--auto-match} option`